bentoml/service.py

```diff
@@ -553,6 +574,21 @@ def artifacts(self):
     def env(self):
         return self._env
 
+    @property
+    def web_static_content(self):
+        return self._web_static_content
+
+    @property
+    def get_web_static_content_path(self):
```

Comment:
usually `property` won't have the verb `get_`,  I feel making this a method instead of a property would be better